lib/libUPnP/Platinum/Source/Devices/MediaServer/PltMediaServer.cpp

```diff
@@ -400,7 +400,8 @@ PLT_MediaServer::ParseTagList(const NPT_String& updates, NPT_Map<NPT_String,NPT_
     for (NPT_List<NPT_XmlNode*>::Iterator children = didl_partial->GetChildren().GetFirstItem(); children; children++) {
         NPT_XmlElementNode* child = (*children)->AsElementNode();
         if (!child) continue;
-        tags[child->GetTag()] = *child->GetText();
```

Comment:
Would you mind splitting this into a separate commit as it's a fundamental fix independent of any Kodi logic.